Model/Util/Service/LocalAuthenticationService.swift

```diff
@@ -18,6 +18,28 @@ class LocalAuthenticationService {
         var error: NSError?
         let description: String = "認証します"
 
+        return Observable.create { observable in
+            if context.canEvaluatePolicy(.deviceOwnerAuthentication, error: &error) {
+                context.evaluatePolicy(.deviceOwnerAuthentication, localizedReason: description, reply: {success, _ in
+                    if (success) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/eifandevs/Amby/blob/13b22eabc34a3da4be6d8d051199f3a006a4b490/Model/Util/Service/LocalAuthenticationService.swift#L24"></span><code>if</code>, <code>for</code>, <code>guard</code>, <code>switch</code>, <code>while</code>, and <code>catch</code> statements shouldn’t unnecessarily wrap their conditionals or arguments in parentheses.<br />
<code>control_statement</code> <code>LocalAuthenticationService.swift:24</code></td>
    </tr>
  </tbody>
</table>